dali/test/python/test_external_source_pytorch_cpu.py

```diff
@@ -15,7 +15,8 @@
 # it is enough to just import all functions from test_internals_operator_external_source
 # nose will query for the methods available and will run them
 # the test_internals_operator_external_source is 99% the same for cupy and numpy tests
-# so it is better to store everything in one file and just call `use_cupy` to switch between the default numpy and cupy
+# so it is better to store everything in one file and just call `use_cupy`
+# to switch between the default numpy and cupy
 
-from test_external_source_impl import *
-use_torch(False)
+from test_external_source_impl import *  # noqa: F403
+use_torch(False)  # noqa: F405
```

Comment:
Could it be like this?
```suggestion
from test_external_source_impl import use_torch
use_torch(False)  # noqa: F405
```

Reply:
No. This file needs to import all test functions from the test_external_source_impl module. Above those lines there's comment that explains this